sound/soc/sof/sof-audio.c

```diff
@@ -433,6 +433,26 @@ struct snd_sof_dai *snd_sof_find_dai(struct snd_soc_component *scomp,
 	return NULL;
 }
 
+/*
+ * Helper to get SSP MCLK from a pcm_runtime.
+ * Return 0 if not exist.
+ */
+int snd_sof_dai_get_ssp_mclk(struct snd_soc_pcm_runtime *rtd)
+{
+	struct snd_soc_component *component =
+		snd_soc_rtdcom_lookup(rtd, SOF_AUDIO_PCM_DRV_NAME);
```

Comment:
@keyonjie this looks good but I think the snd_soc_rtdcom_lookup() function has been proposed to be removed or modifed to support multi-cpu/codecs. Can we use asoc_rtd_to_codec(rtd, 0) to get the codec DAI and the derive the component from there instead?

Reply:
> @keyonjie this looks good but I think the snd_soc_rtdcom_lookup() function has been proposed to be removed or modifed to support multi-cpu/codecs. Can we use asoc_rtd_to_codec(rtd, 0) to get the codec DAI and the derive the component from there instead?

Thanks for the information, do you mean asoc_rtd_to_cpu(rtd, 0) actually? I am not familiar with that but all our other SOF parts are using snd_soc_rtdcom_lookup(), how about leave it as it is for consistence, and change it together in future if needed?

Reply:
sure I guess thats fine for now.